services/authentication/src/main/java/com/wedriveu/services/authentication/entity/UserStoreImpl.java

```diff
@@ -71,10 +75,14 @@ private void writeJSonUsersFile(ArrayList<User> userListToJSon){
         }
     }
 
-    private User getUserRequestedCheckingUsersList(User[] users, String username){
+    private User getUserRequestedCheckingUsersList(List<User> users, String username){
         for (User user : users) {
             if(user.getUsername().equals(username)) {
-                Log.log("Login success! " + "Username: " + user.getUsername() + ", Password: " + user.getPassword());
+                Log.log("Login success! " +
```

Comment:
I think that is useful for debugging. Maybe, unless we merge all the components and we're sure that they work perfectly, we can use these logs.